modules/NEON/README.md

```diff
@@ -15,7 +17,7 @@ This RAD Lab module provides a managed environment for research and data analyti
 
 Below Architechture Diagram is the base representation of what will be created as a part of [RAD Lab Launcher](../../radlab-launcher/radlab.py).
 
-![](../../docs/images/V9_NEON.png)
+![](./images/architecture.png)
```

Comment:
This file is not present.

Reply:
Looks like it got put in a new folder `neon/images/` instead of `NEON/images/`

Reply:
Resolved